### ISSUE TITLE
make protobuf marshal `big.Int` and `[]big.int`

### ISSUE THREAD

**Issue by decltypeme** [dedis/cothority#1590]
Need to implement support for marshalling/unmarshalling of big.Int and arrays of big.Ints

**Comment by decltypeme** [comment]
so big.Int has two private data members
```
	neg bool // sign
	abs nat  // absolute value of the integer
```
and we have a strict check in protobuf for public members
```
	if noPublicFields {
		panic("struct has no serializable fields")
	}
```
that’s why, this is cause. I am curious though why this check is necessary?

**Comment by jeffallen** [comment]
dedis/protobuf uses reflection to walk the contents of the data structure and emit the protobuf-encoded data. Having private fields means that we cannot serialise it from "outside" the package. This is why we need to use the public API of big.Int in order to serialise it.

**Comment by jeffallen** [comment]
See issue dedis/protobuf#54 and fix dedis/protobuf#55.
